flink-shaded-hadoop/pom.xml

```diff
@@ -111,6 +111,7 @@ under the License.
 									<include>io.netty:netty:*</include>
 									<include>org.apache.curator:*</include>
 									<include>org.apache.hadoop:*</include>
+									<include>org.mortbay.jetty:*</include>
```

Comment:
The `jetty-util` will actually be included in the Flink fat jar without this line. This line simply adds the `jetty-util` jar in addition into the `flink-shaded-hadoop1.jar` which is an intermediate artifact, but also deployed to maven. I would like to keep the dependency out of the intermediate artifact.